default.conf

```diff
@@ -1,7 +1,10 @@
 server {
     listen 80;
     server_name _;
-
+    root   /usr/share/nginx/html;
+    index  index.html index.htm;
+    
+    #Cache the static content for 2 days
```

Comment:
```suggestion
    # Cache the static content for 2 hours
```

Reply:
if possible can we remove this general header and target specific files
```
    location ~* \.(?:js|css)$ {
        add_header Cache-Control "public, max-age=7200, immutable";
    }
```

Reply:
We actually target the specific files & folders to not cache them. So, we can just cache the remaining files as the filenames have content hash in them.
If we are going to target specific file extensions to cache, then we should add `svg|png` too and ignore these files in `/public/images` and `/media`